lib/types/cards.d.ts

```diff
@@ -1,6 +1,42 @@
 import { Payments } from "./payments";
 import { INormalizeError } from "./api";
 
+export declare namespace Cards {
+  interface RazorpayCardReferenceNumberBaseRequest {
+    /**
+     *  The tokenised card number whose PAR or network reference id should be retrieved.
+     */
+    number: string;
+    /**
+     * Determines if the card is saved as a token. possible value is `true` or `false`
+     */
+    tokenised?: boolean;
+  }
+
+  interface RazorpayCardReferenceTokenBaseRequest {
+    /**
+     * The token whose PAR or network reference id should be retrieved.
+     */
+    token: string;
+  }
+
+  interface RazorpayCardReference {
+    /**
+     *  The card network.
+     */
+    network?: Network;
+    payment_account_reference: string | null;
```

Comment:
`payment_account_reference` is a mandatory field that accepts `null` ? Or is this an optional field?

Reply:
This interface is for response, so the user will get either null or some value in it

Reply:
Got it